pkg/backend/httpstate/snapshot.go

```diff
@@ -102,23 +101,10 @@ func (persister *cloudSnapshotPersister) saveDiff(ctx context.Context,
 }
 
 func (persister *cloudSnapshotPersister) saveFullVerbatim(ctx context.Context,
-	differ *deploymentDiffState, deployment *apitype.UntypedDeployment, token string) error {
```

Comment:
Nice! This really cleans up the function by encapsulating the logic in `marshalUntypedDeployment`